src/ChatSystemServiceProvider.php

```diff
@@ -59,7 +59,11 @@ function bootForConsole(): void {
     // Publishing the migration file.
     if (! class_exists('CreateConversationsTable')) {
         $this->publishes([
-            __DIR__.'/../database/migrations' => database_path('migrations'),
+            __DIR__.'/../database/migrations/create_conversations_table.php.stub' => database_path('migrations/' . date('Y_m_d_His', time()) . '_create_conversations_table.php'),
+            __DIR__.'/../database/migrations/create_conversation_users_table.php.stub' => database_path('migrations/' . date('Y_m_d_His', time()) . '_create_conversation_users_table.php'),
+            __DIR__.'/../database/migrations/create_messages_table.php.stub' => database_path('migrations/' . date('Y_m_d_His', time()) . '_create_messages_table.php'),
+            __DIR__.'/../database/migrations/create_chat_events_table.php.stub' => database_path('migrations/' . date('Y_m_d_His', time()) . '_create_chat_events_table.php'),
+            __DIR__.'/../database/migrations/create_metas_table.php.stub' => database_path('migrations/' . date('Y_m_d_His', time()) . '_create_metas_table.php'),
```

Comment:
By renaming the migration files.
every time you run `vendor:publis --tag='config'` will always create new migration files which created duplicate files.

having static files names will prevent copying migration files multiple times.

so it think it would be better not to rename the migration files.

Example duplicate:
![Screenshot 2021-06-28 at 20 52 08](https://user-images.githubusercontent.com/34090541/123696074-3f405d00-d853-11eb-978e-f697cb17e7f4.png)

Reply:
And always don't use your fork master branch to send pull request.
try create a new branch for your changes.
so that your master could be in sync with upstreams branch.